nuxt.config.js

```diff
@@ -1,3 +1,5 @@
+require('dotenv').config();
```

Comment:
Can't you use ESM inside the Nuxt config?

Reply:
Switched to use ESM